### ISSUE TITLE
Design a custom 404 page

### ISSUE THREAD

**Issue by marcominerva** [marcominerva/TinyCMS#15]
It is necessary to provide an attractive example of 404 page.

**Comment by marcominerva** [comment]
To implement this layout I think that custom CSS support (#11) is required. Would you like to take care of this issue? ;)

**Comment by andreatosato** [comment]
Do you want static image?

**Comment by marcominerva** [comment]
Just a simple static page with a nice layout, you can embed a Base64 image if you mind creating something like this: https://cdn.searchenginejournal.com/wp-content/uploads/2020/08/killer-404-page-disney-5f3d563a47cff.png.

And then, add a `scripts` folder in the `database` folder with the correspoding query to add it to the database (you must get the ID of the first site, because we currently support only a single site, and then use it to insert the page in the **ContentPages** table).

**Comment by andreatosato** [comment]
I must create sql insert for first site, so I can add or update this content page. Ok?
> you must get the ID of the first site, because we currently support only a single site


Can I add SVG + CSS ?
[Example](https://codepen.io/gxash/pen/MjQOom)

Static base64 has scale problem and resolution

**Comment by marcominerva** [comment]
Yes, you can create a general **init.sql** script that creates the site and the content page. At his point, provide also a simple Home Page (just a simple title is fine), so the script can be used to create a first site from scratch.

Yes, the CMS provides support for embedding CSS directly in the ContentPage definition.

**Comment by andreatosato** [comment]
Can you create another issue for "init script"?
So I can create 2 parallel branch for two problem

**Comment by marcominerva** [comment]
You can find it here: #26
